pkgs/development/compilers/tvm/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchgit, cmake, git }:
+stdenv.mkDerivation rec {
+  pname = "tvm";
+  version = "0.7.0";
+  src = fetchgit {
```

Comment:
Use `fetchFromGitHub` instead.

Reply:
Happy to make the change, may I ask the reasoning? Doesn't `fetchFromGitHub` delegate to fetchgit`, the primary differences being ability to clone private repos (not relevant in this case) and not doing a submodule clone by default?

Reply:
`fetchFromGitHub` uses `fetchzip` under the hood in such case, that's much faster.
https://github.com/NixOS/nixpkgs/blob/f2d8aebe8e597a2ba9b13769fbb0fcaa06b14b16/pkgs/build-support/fetchgithub/default.nix#L14

Reply:
Ah in this case we do need to `fetchSubmodules` so it would be identical to `fetchgit` right?

Reply:
Yes, but anyway `fetchFromGitHub` is more appropriate function for packages from GitHub.

Reply:
Fair enough, done!